libs/device-core/src/commands/entities/AppStorageInfo.ts

```diff
@@ -0,0 +1,7 @@
+export type AppStorageInfo = {
+  size: number;
+  version: string;
```

Comment:
I would call this `dataVersion` to avoid any confusion with the app's version, if I recall correctly it's a different thing.